src/domains/ecommerce-common/02_security.tf

```diff
@@ -486,6 +486,19 @@ resource "azurerm_key_vault_secret" "transactions_service_auth_update_api_key" {
   }
 }
 
+
+resource "azurerm_key_vault_secret" "user_stats_api_key" {
+  name         = "user-stats-api-key"
```

Comment:
user-stats is an ecommerce microservice, who will invoke this api? I recommend adding a reference to the client in the name, the goal is to avoid using this api key unintentionally. If there are multiple clients internal or external to ecommerce, it is always better to differentiate the api keys